packages/babel-preset-default/README.md

```diff
@@ -12,26 +14,27 @@ npm install @wordpress/babel-preset-default --save-dev
 
 ### Usage
 
-#### Via .babelrc (Recommended)
+There are a number of methods to configure Babel. See [Babel's Configuration documentation](https://babeljs.io/docs/en/configuration) for more information. To use this preset, simply reference `@wordpress/default` in the `presets` option in your Babel configuration.
+
+For example, using `.babelrc`:
 
 ```json
 {
   "presets": [ "@wordpress/default" ]
 }
 ```
 
-#### Via CLI
+#### Extending Configuration
 
-```bash
-babel script.js --presets @wordpress/default
-```
+This preset is an opinionated configuration intended to support the Gutenberg coding guidelines. If you would like to add to or change this configuration, you can do so by expanding your Babel configuration to include plugins or presets which override those included through this preset. It may help to familiarize yourself [the implementation of the configuration](https://github.com/WordPress/gutenberg/blob/master/packages/babel-preset-default/index.js) to see which specific plugins are enabled by default through this preset.
```

Comment:
One thing, should we replace "Gutenberg` with "WordPress packages" or something like that?

Reply:
> One thing, should we replace "Gutenberg` with "WordPress packages" or something like that?

Probably? My thinking at the time was that the stance here about new JavaScript features is one which really only applies to this repository at the moment, not the WordPress project generally. Maybe your addition of "packages" would be enough of a middle ground here, since I think it's fair to call this repository as the canonical source of WordPress packages.

Reply:
It's more nuanced. Gutenberg uses 2 more plugins:

https://github.com/WordPress/gutenberg/blob/8fd9332c1588967d8ea4f0af94058045f20c9654/babel.config.js#L6

Whatever you decide is fine :)

Reply:
I arrived at the best solution: Remove it! 😆 6c50156

I think it should be okay. There is an exhaustive explanation in the introduction paragraph about what exactly this includes, and a link to the coding guidelines. It should be fine at this point in the documentation to just say "it's opinionated" and not repeat the details.